### ISSUE TITLE
Database import feature

### ISSUE THREAD

**Issue by hirnsturm** [jackd248/db-sync-tool#4]
### Feature Request

Some servers needs special handeling before scripts can execute by SSH. Is it possible to offer before_script and after_script functionality to add additional shell commands? Those should execute before or after db-sync-tool commands.

The configuration might extended like this:
```json
{
  "name": "",
  "type": "",
  "target": {
    "before_script": "",
    "after_script": ""
  },
  "origin": {
    "before_script": "",
    "after_script": ""
  },
  "ignore_table": [
  ]
}

```

**Comment by jackd248** [comment]
Feature is implemented in version **1.5.0**.

**Comment by arislazuardi** [comment]
Is there an option to allow tables? Because the requirement is only for a few tables out of hundreds of tables.
thks
